pom.xml

```diff
@@ -143,6 +143,16 @@
         <artifactId>dropwizard-core</artifactId>
         <version>${dropwizard.version}</version>
       </dependency>
+      <dependency>
+        <groupId>com.restfb</groupId>
+        <artifactId>restfb</artifactId>
+        <version>${restfb.version}</version>
```

Comment:
You need to define `restfb.version` above so that this pulls in the right version.